src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -360,11 +385,26 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
             </TokenData>
           )}
         </TokenInfo>
-        {listItems?.length > 1 && displayGuidance && (
-          <ZapMessageContainer onClick={openSearchList}>
-            ⚡ {t('components.transaction.zap-guidance.part-1')}{' '}
-            <HighlightText> {t('components.transaction.zap-guidance.part-2')} </HighlightText>{' '}
-            {t('components.transaction.zap-guidance.part-3')}
+
+        {zappableItems?.length > 1 && displayGuidance && (
+          <ZapMessageContainer>
+            {t('components.transaction.zap-guidance.desc')}
+            <ZappableTokensList>
+              {zappableItems.slice(0, 4).map((item) => (
```

Comment:
we are slicing `zappableItems` again here, we should just do once (4 or 5). Maybe use `const SHORTCUT_DISPLAY_TOKENS = 4;`